superbench/common/utils/stdout_logging.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""SuperBench stdout logging module."""
+
+import sys
+
+
+class SuperBenchStdoutLogger:
+    """Logger class to enable or disable to redirect STDOUT and STDERR to file."""
```

Comment:
rename to StdLogger? seems it also redirects stderr

Reply:
updated